<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FishBot配置工具(VCODE) fishros.org.cn </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>781</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>2.固件烧录</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>60</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>固件地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="binAddress">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>681</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="downloadButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>一键下载</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>521</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>当前默认最新固件|其他获取固件地:https://fishros.org.cn/forum/topic/368</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>1.设备选择</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>60</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>设备端口</string>
     </property>
    </widget>
    <widget class="QComboBox" name="devicesComboBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>541</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>请选择设备端口号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="freshDevicePortButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
    <widget class="QComboBox" name="deviceTypeComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>541</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>FishBot主控板|MicroROS学习板</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FishBot雷达转接板</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FishBot四驱板|MicroROS四驱板</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FishBot Camera无线摄像头</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>60</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>设备类型</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restartDeviceButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>重启设备</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>791</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>系统日志</string>
    </property>
    <widget class="QTextEdit" name="system_log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>771</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>781</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>3.配置设备</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>68</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>配置项</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>68</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>配置值</string>
     </property>
    </widget>
    <widget class="QComboBox" name="configKeyComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>541</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="configValueLineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>541</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="configButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>90</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>一键配置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scanConfigButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>重新扫描配置</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_fishros"/>
   </widget>
   <widget class="QMenu" name="menu_shop">
    <property name="title">
     <string>官方店铺</string>
    </property>
    <addaction name="action_shop"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>快捷操作</string>
    </property>
    <addaction name="action_flash_and_config"/>
   </widget>
   <addaction name="menu_about"/>
   <addaction name="menu_shop"/>
   <addaction name="menu"/>
  </widget>
  <action name="action_fishros">
   <property name="text">
    <string>鱼香ROS</string>
   </property>
  </action>
  <action name="action_shop">
   <property name="text">
    <string>官方店铺</string>
   </property>
  </action>
  <action name="action_flash_and_config">
   <property name="text">
    <string>一键下载并写入配置(F8)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
